mpas_analysis/sea_ice/modelvsobs.py

```diff
@@ -183,11 +193,11 @@ def seaice_modelvsobs(config, streamMap=None, variableMap=None):
         months = monthsInClim[climName]
         month = months[0]
         days = daysInMonth[month-1]
-        climatology = days*monthlyClimotology.sel(month=month)
+        climatology = days*monthlyClimatology.sel(month=month)
         totalDays = days
         for month in months[1:]:
             days = daysInMonth[month-1]
-            climatology += days*monthlyClimotology.sel(month=month)
+            climatology += days*monthlyClimatology.sel(month=month)
             totalDays += days
         climatology /= totalDays
```

Comment:
I don't think it's very helpful to note these things here because they'll be forgotten as soon as this PR gets merged.  You'd need to create issues for them instead.